cli/lib/kontena/cli/nodes/show_command.rb

```diff
@@ -26,7 +26,7 @@ def execute
       puts "  kernel: #{node['kernel_version']}"
       puts "  drivers:"
       puts "    storage: #{node['driver']}"
-      puts "    volume: #{node['volume_drivers'].to_a.map{|v| v['name']}.join(',')}"
+      puts "    volume: #{node['volume_drivers'].to_a.map{|v| [v['name'], v['version']].compact.join(':')}.join(',')}"
```

Comment:
Imho we should not expose plugin version here to user. It's misleading in too many ways.